Logging started at 2016-07-16 19:17:20
File name: C:\Users\Administrator\Dropbox\lunarisle source\.\data\logs\err\1468689440039 - Administrator - [Sat Jul 16][7.17PM].txt
System.getProperty("os.name"): "Windows Server 2012 R2"

[2016-07-16 19:17:39] java.lang.NumberFormatException: For input string: ""
[2016-07-16 19:17:40] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-16 19:17:40] 	at java.lang.Integer.parseInt(Unknown Source)
[2016-07-16 19:17:40] 	at java.lang.Integer.parseInt(Unknown Source)
[2016-07-16 19:17:40] 	at com.vencillio.core.util.FileHandler.loadMaxPlayers(FileHandler.java:52)
[2016-07-16 19:17:40] 	at com.vencillio.core.util.FileHandler.load(FileHandler.java:24)
[2016-07-16 19:17:40] 	at com.vencillio.GameDataLoader.load(GameDataLoader.java:185)
[2016-07-16 19:17:40] 	at com.vencillio.core.GameThread.startup(GameThread.java:69)
[2016-07-16 19:17:40] 	at com.vencillio.core.GameThread.init(GameThread.java:34)
[2016-07-16 19:17:40] 	at com.vencillio.Server.main(Server.java:104)
[2016-07-16 19:20:57] java.lang.NullPointerException
[2016-07-16 19:20:57] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-16 19:20:57] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-16 19:20:57] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-16 19:20:57] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 19:20:57] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 19:20:57] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 19:20:57] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 19:20:57] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:16:01] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-16 21:16:01] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:16:01] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:16:01] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:16:01] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:16:01] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:16:01] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:16:01] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:16:42] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-16 21:16:42] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:16:42] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:16:42] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:16:42] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:16:42] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:16:42] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:16:42] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:44:58] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:44:58] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:44:58] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:44:59] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:44:59] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:44:59] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:44:59] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:44:59] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:45:01] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:45:01] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:45:01] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:45:01] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:45:01] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:45:01] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:45:01] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:45:01] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:45:04] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:45:04] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:45:04] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:45:04] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:45:04] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:45:04] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:45:04] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:45:04] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:45:07] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:45:07] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:45:07] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:45:07] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:45:07] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:45:07] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:45:07] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:45:07] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:45:11] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:45:11] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:45:11] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:45:11] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:45:11] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:45:11] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:45:11] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:45:11] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:45:15] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:45:15] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:45:15] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:45:15] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:45:15] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:45:15] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:45:15] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:45:15] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-16 21:45:18] java.lang.ArrayIndexOutOfBoundsException: 249
[2016-07-16 21:45:18] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-16 21:45:18] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-16 21:45:18] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-16 21:45:18] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-16 21:45:18] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-16 21:45:18] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-16 21:45:18] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 00:12:42] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:12:42] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-17 00:12:42] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-17 00:12:42] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-17 00:12:42] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-17 00:12:42] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-17 00:12:42] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-17 00:12:42] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-17 00:12:42] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-17 00:12:42] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-17 00:12:42] 	at java.lang.Thread.run(Unknown Source)
[2016-07-17 00:12:42] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:12:42] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-17 00:12:42] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-17 00:12:42] 	... 9 more
[2016-07-17 00:12:42] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:12:42] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-17 00:12:42] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-17 00:12:42] 	... 10 more
[2016-07-17 00:12:42] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:12:42] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-17 00:12:42] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-17 00:12:42] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-17 00:12:42] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-17 00:12:42] 	... 11 more
[2016-07-17 00:13:45] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:13:45] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-17 00:13:45] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-17 00:13:45] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-17 00:13:45] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-17 00:13:45] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-17 00:13:45] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-17 00:13:45] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-17 00:13:45] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-17 00:13:45] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-17 00:13:45] 	at java.lang.Thread.run(Unknown Source)
[2016-07-17 00:13:45] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:13:45] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-17 00:13:45] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-17 00:13:45] 	... 9 more
[2016-07-17 00:13:45] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:13:45] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-17 00:13:45] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-17 00:13:45] 	... 10 more
[2016-07-17 00:13:45] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-17 00:13:45] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-17 00:13:45] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-17 00:13:45] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-17 00:13:45] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-17 00:13:45] 	... 11 more
[2016-07-17 00:20:34] java.lang.NumberFormatException: For input string: "6.5000000"
[2016-07-17 00:20:34] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-17 00:20:34] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-17 00:20:34] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-17 00:20:34] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:208)
[2016-07-17 00:20:34] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-17 00:20:34] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 00:20:34] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 00:20:34] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 00:20:34] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 00:20:34] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 01:13:30] java.lang.NullPointerException
[2016-07-17 01:13:30] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-17 01:13:30] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-17 01:13:30] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 01:13:30] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 01:13:30] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 01:13:30] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 01:13:30] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 01:13:49] java.lang.NullPointerException
[2016-07-17 01:13:49] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-17 01:13:49] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-17 01:13:49] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 01:13:49] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 01:13:49] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 01:13:49] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 01:13:49] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 03:31:09] java.lang.NullPointerException
[2016-07-17 03:31:09] 	at com.vencillio.rs2.content.combat.formula.RangeFormulas.calculateRangeAttack(RangeFormulas.java:78)
[2016-07-17 03:31:09] 	at com.vencillio.rs2.content.combat.impl.Ranged.execute(Ranged.java:41)
[2016-07-17 03:31:09] 	at com.vencillio.rs2.content.combat.Combat.attack(Combat.java:92)
[2016-07-17 03:31:09] 	at com.vencillio.rs2.content.combat.Combat.process(Combat.java:215)
[2016-07-17 03:31:09] 	at com.vencillio.rs2.entity.player.Player.process(Player.java:1561)
[2016-07-17 03:31:09] 	at com.vencillio.rs2.entity.World.process(World.java:322)
[2016-07-17 03:31:09] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 03:31:09] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 03:51:04] java.lang.NumberFormatException: For input string: "all"
[2016-07-17 03:51:04] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-17 03:51:04] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-17 03:51:04] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-17 03:51:04] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:208)
[2016-07-17 03:51:04] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-17 03:51:04] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 03:51:04] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 03:51:04] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 03:51:04] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 03:51:04] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 04:55:34] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-17 04:55:34] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-17 04:55:34] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-17 04:55:34] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 04:55:34] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 04:55:34] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 04:55:34] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 04:55:34] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 04:55:38] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-17 04:55:38] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-17 04:55:38] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-17 04:55:38] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 04:55:38] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 04:55:38] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 04:55:38] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 04:55:38] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 17:14:36] java.lang.NullPointerException
[2016-07-17 17:14:36] 	at com.vencillio.rs2.entity.player.net.in.impl.ReceiveString.handle(ReceiveString.java:56)
[2016-07-17 17:14:36] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 17:14:36] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 17:14:36] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 17:14:36] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 17:14:36] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 17:15:25] java.lang.NullPointerException
[2016-07-17 17:15:25] 	at com.vencillio.rs2.entity.player.net.in.impl.ReceiveString.handle(ReceiveString.java:56)
[2016-07-17 17:15:25] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 17:15:25] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 17:15:25] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 17:15:25] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 17:15:25] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 17:16:09] java.lang.NullPointerException
[2016-07-17 17:16:09] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-17 17:16:09] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-17 17:16:09] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 17:16:09] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 17:16:09] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 17:16:09] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 17:16:09] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-17 17:16:28] java.lang.NullPointerException
[2016-07-17 17:16:28] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-17 17:16:28] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-17 17:16:28] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-17 17:16:28] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-17 17:16:28] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-17 17:16:28] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-17 17:16:28] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
